src/roller.js

```diff
@@ -111,12 +111,21 @@ class LMRTFYRoller extends Application {
             fakeEvent.shiftKey = false;
             fakeEvent.altKey = false;
             fakeEvent.ctrlKey = true;
+        } else if (this.advantage === 2) {
+            fakeEvent.shiftKey = LMRTFY.defaultQuery;
+            fakeEvent.altKey = false;
+            fakeEvent.ctrlKey = false;
         }
         const rollMode = game.settings.get("core", "rollMode");
         game.settings.set("core", "rollMode", this.mode || CONST.DICE_ROLL_MODES);
         for (let actor of this.actors) {
             Hooks.once("preCreateChatMessage", this._tagMessage.bind(this));
-            actor[rollMethod].call(actor, ...args, { event: fakeEvent });
+            if(game.system.id=="pf2e") {
+                actor[rollMethod].call(actor, fakeEvent, ...args);                        
+            } else {
+                actor[rollMethod].call(actor, ...args, { event: fakeEvent });                        
```

Comment:
I hadn't noticed this when I reviewed last time. Weird... Oh well, I'll blame it on me being senile!

Reply:
Yeah, not sure what to do about this one:
PF2:   rollAbility(event, abilityName) {
  5E:   rollAbility(abilityId, options={}) {

What do you suggest?

Reply:
No suggestion, looks good, I was just surprised that I hadn't noticed this change in my previous review.